tests/Integration/Monitoring/Test_Plotter.py

```diff
@@ -0,0 +1,408 @@
+                                                   1562848200: 260.68917410714283,
+                                                   1562859900: 0,
+                                                   1562850900: 180.12890625,
+                                                   1562851800: 186.35546875,
+                                                   1562847300: 121.13671875,
+                                                   1562852700: 186.35807291666666}
+                         },
+                'unit': 'MB',
+                'granularity': 900}
+
+    obj = ComponentMonitoringPlotter(None, None)
+    res = obj._plotMemoryUsage(reportRequest, plotInfo, plotName)
+    self.assertEqual(res['OK'], True)
+    self.assertEqual(res['Value'], {'plot': True, 'thumbnail': False})
+
+    res = compare('%s.png' % plotName, '/home/dirac/DIRAC/tests/Integration/Monitoring/png/%s.png' % plotName)
+    self.assertEqual(0.0, res)
+
+  def test_plotRunningTime(self):
+    """ test for the method "_plotRunningTime"
+    """
+
+    plotName = "ComponentMonitoringPlotter_plotRunningTime"
+    reportRequest = {'grouping': 'component',
+                     'groupingFields': ('%s', ['component']),
+                     'startTime': 1562796000,
+                     'endTime': 1562882400,
+                     'condDict': {'component': 'Monitoring_Monitoring'}
+                     }
+
+    plotInfo = {'data': {'Monitoring_Monitoring': {1562853600: 313.32080459594727,
+                                                   1562857200: 0,
+                                                   1562854500: 0,
+                                                   1562855400: 0,
+                                                   1562856300: 0,
+                                                   1562858100: 0,
+                                                   1562849100: 1754.9678344726562,
+                                                   1562860800: 0,
+                                                   1562850000: 492.68797302246094,
+                                                   1562848200: 854.48386492048,
+                                                   1562859900: 0,
+                                                   1562850900: 1332.7680140904017,
+                                                   1562851800: 1812.8760986328125,
+                                                   1562847300: 314.9701385498047,
+                                                   1562852700: 3452.56787109375}
+                         },
+                'unit': 'seconds',
+                'granularity': 900}
+
+    obj = ComponentMonitoringPlotter(None, None)
+    res = obj._plotRunningTime(reportRequest, plotInfo, plotName)
+    self.assertEqual(res['OK'], True)
+    self.assertEqual(res['Value'], {'plot': True, 'thumbnail': False})
+
+    res = compare('%s.png' % plotName, '/home/dirac/DIRAC/tests/Integration/Monitoring/png/%s.png' % plotName)
```

Comment:
I am pretty sure @zmathe told you multiple times not to hardcode path like that

Reply:
Like we had discussed on slack he asked me to keep the path as `/home/dirac`.

Reply:
No, he told you that you cannot have hardcoded path, because it can be anywhere, for example `/home/dirac` instead of `/opt/dirac`

Reply:
@pujanm I told you many times do not hardcode  the path. You can see in the screen shoot:
![Screenshot from 2019-07-31 14-54-51](https://user-images.githubusercontent.com/981448/62213522-66f6c400-b3a3-11e9-89fb-a5db50555723.png)

Reply:
Okay sorry @zmathe @chaen will do the change.